tests/nfs/nfs_operations.py

```diff
@@ -1315,3 +1315,113 @@ def nfs_log_parser(client, nfs_node, nfs_name, expect_list=None):
                 log.info(
                     "Since we are collecting logs, ignoring the exception will not fail the test"
                 )
+    return 0
+
+
+def verify_ops_control_settings(client, cluster_name, export_path=None):
+    """Verify ops control settings at cluster and export level"""
+    try:
+        # Verify cluster settings
+        cluster_settings = json.loads(
+            client.exec_command(cmd=f"ceph nfs cluster qos get {cluster_name}")[0]
+        )
+        log.info(f"Current cluster ops control settings: {cluster_settings}")
+        # Verify export settings if path provided
+        if export_path:
+            export_settings = json.loads(
+                client.exec_command(
+                    cmd=f"ceph nfs export qos get {cluster_name} {export_path}"
+                )[0]
+            )
+            log.info(
+                f"Current export ops control settings for {export_path}: {export_settings}"
+            )
+            return cluster_settings, export_settings
+        return cluster_settings, None
+    except Exception as e:
+        log.error(f"Failed to get ops control settings: {str(e)}")
+        return None, None
+
+
+def extract_dd_time(dd_output):
```

Comment:
@SinghSamosa A suggestion here, since its a generic operation, can we move it outside of this file? Maybe under utils? 
So that we can keep nfs specific ops in this file